packages/gatsby-theme-api-docs/src/components/resource/method/query-parameters.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import Parameters from './parameters';
+
+function transformQueryParameterDescriptions(queryParameters) {
+  return queryParameters.map((parameter) => {
+    if (
+      parameter.description &&
+      parameter.type === 'array' &&
+      parameter.items
+    ) {
+      return {
+        ...parameter,
+        description: `${parameter.description}\n\nMultiple instances can be passed. E.g. \`${parameter.name}=<${parameter.items.type}>&${parameter.name}=<${parameter.items.type}>&${parameter.name}=<${parameter.items.type}>\``,
```

Comment:
I suggest to give the hint about multiple instances right in the beginning and then append the regular description afterwards. This way we can explain the semantics of using the parameter several times, like for `where` it is a conjunction, but for `sort` it leads to a multi-sort behavior. Not sure, I did the change correct in the below suggestion, untested 😉 

```suggestion
        description: `The parameter can be passed multiple times, e.g. \`${parameter.name}=<${parameter.items.type}>&${parameter.name}=<${parameter.items.type}>&${parameter.name}=<${parameter.items.type}>\`\n${parameter.description}\n`,
```

Reply:
I prefer stefan's wording "can be passed multiple times" too. 

Concerning the example I am not sure whether it's a good idea or just more confusing (for example, are the `<` placeholders or a necessary part of the syntax?).  

Everywhere else in the documentation we are assuming knowledge on how to build a valid URL query string (not explaining for example that the first one is after a question mark and then separated by ampersands) so I am not sure why we need to given an example of how a query string looks like here.  

Unless you have already discussed why you need the generated example, the following would suffice for me: 
`*This parameter can be passed mulitple times.*\n${parameter.description}`